andela/wger_akira
# Ft cache nutritional values 142465437

#### What does this PR do?
Cache nutritional values for plan
#### Description of Task to be completed?
Cache the nutritional_info dictionary in Nutrition Plan.
#### How should this be manually tested?
Check the time the browser takes to load the Nutrition plan for the first time and when you reload the second time
#### What are the relevant pivotal tracker stories?
 #142465437
